data/dapps/silo-finance.json

```diff
@@ -0,0 +1,6 @@
+{
+  "aliases": {
+    "silo-finance-v1-rdnt": { "chains": ["arbitrum"], "title": "Silo Finance RDNT market" }
```

Comment:
According to [Defillama](https://defillama.com/protocol/silo-finance), the dApp name is `Silo Finance` but it has two versions (for which Defillama has subpages).

For maximum clarity I derived the dApp alias from the dApp name, the version of the protocol and the market asset.